duckduckgo/Android
# Make icons resilient so they can be used without issue across devices

Task/Issue URL: https://app.asana.com/0/361428290920652/1174457371356692

**Description**:
Currently when we use an icon we need to also add a tint to the layout or style for low end devices. This is often omitted which can cause color glitches.

Thus in this task we:
* update the toolbar svgs to use a color attribute (rather than selector with attributes)
* add a selector (without inner attributes) for each theme to the styles that require it
* remove tint from overflow menu icon as no longer needed
* clean up names and remove 24dp form toolbar icons for consistency (we only really need to explicitly add dp when there are multiple versions)
* merge duplicate toolbar icons
* delete unused toolbar icons

**Steps to test this PR**:
Repeat the following test across these combinations:
- a recent API and API 21
- light theme and dark theme
- bottom bar experiment versus control

1. Launch to the home screen
1. Check that the icons color looks correct (it should be obvious if they are broken as they go bright pink!)
1. Open the menu and check that the enabled/disabled colors look correct
1. Visit every screen in the app and again check that the colors are correct 


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
